README.md

```diff
@@ -15,15 +15,15 @@ npm create vue@latest
 ```
 
 > [!NOTE]
->  (`@latest` or `@legacy`) MUST NOT be omitted, otherwise `npm` may resolve to a cached and outdated version of the package.
+> (`@latest` or `@legacy`) MUST NOT be omitted, otherwise `npm` may resolve to a cached and outdated version of the package.
 
 Or, if you need to support IE11, you can create a Vue 2 project with:
 
 ```sh
 npm create vue@legacy
 ```
 
-> [!WARNING]
+> [!WARNING]  
```

Comment:
If no spaces are added here, after running `pnpm format`, the content of the next line will be placed behind, causing rendering errors.